cmake/onnxruntime_providers.cmake

```diff
@@ -537,75 +529,62 @@ if (onnxruntime_USE_OPENVINO)
     "${ONNXRUNTIME_ROOT}/core/providers/shared_library/*.cc"
   )
 
-  if (onnxruntime_USE_OPENVINO_BINARY)
+  # Header paths
+  list(APPEND OPENVINO_INCLUDE_DIR_LIST $ENV{INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/include)
+  list(APPEND OPENVINO_INCLUDE_DIR_LIST $ENV{INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/external/tbb/include)
+  list(APPEND OPENVINO_INCLUDE_DIR_LIST $ENV{INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/external/mkltiny_lnx/include)
 
-    # Header paths
-    list(APPEND OPENVINO_INCLUDE_DIR_LIST $ENV{INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/include)
```

Comment:
why is the inference engine include removed

Reply:
As i said please look at the file and not the changes, as the diff is very confusing. I did not remove inference engine include. It should be there in line 533. Don't know why it's failing on windows. Which openvino version are you using ?